app/components/App.js

```diff
@@ -20,7 +20,14 @@ class App extends React.Component {
 }
 
 App.propTypes = {
-  children: PropTypes.object.isRequired
+  children: PropTypes.object.isRequired,
+  showAllHeaderLinks: PropTypes.bool.isRequired
 };
 
-export default connect()(App);
+function mapStatesToProps(state) {
+  return {
+    showAllHeaderLinks: state.reducers.login.loggedIn
```

Comment:
This should be: 
`showAllHeaderLinks: state.login.loggedIn`

Reply:
Let me check, and I will update as necessary.

Reply:
@juampick I double check and I need the .reducers part, in fact, there are other places in the app where it is used in that same way as I did, for instance:
- UserAddPage.js
- UserEditPage.js